src/server/api/index.ts

```diff
@@ -1,12 +1,14 @@
 import { PublicAPI } from "./hono";
 import { initMiddleware } from "./middlewares/init";
 import { registerCompanyRoutes } from "./routes/company";
+// import { registerSessionRoutes } from "./routes/session";
 
 const api = PublicAPI();
 
 api.use("*", initMiddleware());
 
 // RESTful routes for company
 registerCompanyRoutes(api);
+// registerSessionRoutes(api);
```

Comment:
route to test the new auth middleware. this route can be accessed through next-auth cookie and api key.